### PR TITLE
Prepare for v0.11.0 release

### PR DESCRIPTION
**Pull request by lxn2**
This PR prepares for v0.11.0 release:

- Bumps up version number
- Adds updates to README & NEWS

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -1,5 +1,35 @@
 MXNet Change Log
 ================
+## 0.11.0
+- Major Features
```
Comment (sandeep-krishnamurthy) [inline NEWS.md]: Headers probably ? - "##"

File: NEWS.md [hunk 2]
```diff
@@ -1,5 +1,35 @@
 MXNet Change Log
 ================
```
Comment (nswamy) [inline NEWS.md]: News and Readme update should be pushed after the release has been made.

File: NEWS.md [hunk 3]
```diff
@@ -1,5 +1,35 @@
...
+## 0.11.0
```
Comment (nswamy) [inline NEWS.md]: I don't think we can call it 0.11.0 before the vote succeeds. It should be 0.11.0-rc0

File: README.md [hunk 4]
```diff
@@ -22,6 +22,7 @@ deep learning systems, and interesting insights of DL systems for hackers.
 
 What's New
 ----------
+* [Version 0.11.0 Release](https://github.com/dmlc/mxnet/releases/tag/v0.11.0) - MXNet 0.11.0 Release.
```
Comment (nswamy) [inline README.md]: same as above. should be 0.11.0-rc0

File: NEWS.md (hunk 3 above)
Comment (srochel) [inline NEWS.md]: should we include the Keras and CoreML section as well from https://cwiki.apache.org/confluence/display/MXNET/v0.11.0+Release+Notes ?

**Pull request merged**